### PR TITLE
gen_coords now updates vm_params.yaml when generated all related file…

### PR DESCRIPTION
**Pull request by shot0829**
…s (instead of in install simulation step)

### CODE REVIEW

File: SrfGen/gen_coords.py [hunk 1]
```diff
@@ -7,8 +7,9 @@
 
 from qcore import qclogging
 from qcore.binary_version import get_unversioned_bin
-from qcore.constants import VM_PARAMS_FILE_NAME
-from qcore.utils import load_yaml
+from qcore.constants import VM_PARAMS_FILE_NAME, VMParams
```
Comment (jpa198) [inline SrfGen/gen_coords.py:10]: Is VM_PARAMS_FILE_NAME still needed?

**Review by jpa198 — DISMISSED** [review]
Tied in with slurm_workflow PR. Overlapping functionality means no issues with a mismatch (gap between this and that PR going in)

**Review by jasonmotha — APPROVED** [review]

File: SrfGen/gen_coords.py (hunk 1 above)
Comment (sungeunbae) [reply SrfGen/gen_coords.py:10]: @shot0829  I agree - Fix this and I'll approve it.

**Review by sungeunbae — COMMENTED** [review]
See the comment.

**Review by sungeunbae — APPROVED** [review]

**Review by jasonmotha — APPROVED** [review]

**Review by jpa198 — APPROVED** [review]
Why do we need to update vm_params? Is this for adv IMs?

**Comment by shot0829** [comment]
> Why do we need to update vm_params? Is this for adv IMs?

no, this is for basic workflow. when VMs are generated, we need to run the binary 'gen_model_coords' to generate all other files like model_coords, model_bounds etc. and update vm_params to contain those files.
But because of how our old workflow works, the vm_params was only updated once we run install_cybershake, which is not ideal (since the install should ideally not modify any input)

**Pull request merged**